docs/csharp/programming-guide/classes-and-structs/constructors.md

```diff
@@ -14,7 +14,7 @@ Whenever a [class](../../language-reference/keywords/class.md) or [struct](../..
 
 ## Constructor syntax
 
-A constructor is a method whose name is the same as the name of its type. Its method signature includes only the method name and its parameter list; it does not include a return type. The following example shows the constructor for a class named `Person`.
+A constructor is a method whose name is the same as the name of its type. Its method signature includes only an [access modifier](./access-modifiers.md), the method name and its parameter list; it does not include a return type. The following example shows the constructor for a class named `Person`.
```

Comment:
The access modifier is optional (as is true for all methods). We should add that here:

```suggestion
A constructor is a method whose name is the same as the name of its type. Its method signature includes only an optional [access modifier](./access-modifiers.md), the method name and its parameter list; it does not include a return type. The following example shows the constructor for a class named `Person`.
```